pom.xml

```diff
@@ -229,6 +229,10 @@
 			<artifactId>scenery</artifactId>
 			<version>0.1.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>net.imagej</groupId>
+			<artifactId>imagej-legacy</artifactId>
```

Comment:
Nevertheless I'll keep it as optional and continue to stub my toe. Its better than an infection.